.circleci/config.yml

```diff
@@ -42,6 +43,21 @@ jobs:
           docker-username: DOCKER_USERNAME
           docker-password: DOCKER_PASSWORD
           image: honeycombio/honeycomb-opentelemetry-collector
+      - aws-ecr/ecr_login:
+          account_id: "702835727665"
+          public_registry: true
+      - run:
+          name: Build and Push Multi-Arch Docker Image to Public ECR
+          command: |
+            docker buildx build \
+              --platform linux/amd64,linux/arm64 \
+              --sbom=true \
+              --provenance=true \
+              --push \
+              -t public.ecr.aws/honeycombio/honeycomb-opentelemetry-collector:$CIRCLE_SHA1 \
+              -t public.ecr.aws/honeycombio/honeycomb-opentelemetry-collector:$CIRCLE_TAG \
+              -t public.ecr.aws/honeycombio/honeycomb-opentelemetry-collector:latest \
```

Comment:
Can we build the images once and then publish to both docker and ECR like we do for [Refinery](https://github.com/honeycombio/refinery/blob/main/.circleci/config.yml#L183)? Building twice  means it takes more time and we'll end up with different image SHAs between the two repositories.

Reply:
Sure, although Refinery's doesn't guarantee the same image SHA. We'll need to change the Refinery build process if we want that. But did the image build and push once in this repo.